src/migration/1736350167538-UserReport.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserReport1736350167538 implements MigrationInterface {
+    name = 'UserReport1736350167538'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "user_report" ("reportedUserId" text NOT NULL, "userId" character varying(36) NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "reason" character varying(36) NOT NULL, "note" text, CONSTRAINT "PK_f804f389ba2f3ac0313080a1a82" PRIMARY KEY ("reportedUserId", "userId"))`);
+        await queryRunner.query(`CREATE INDEX "IDX_user_report_reported_user_id" ON "user_report" ("reportedUserId") `);
+        await queryRunner.query(`CREATE INDEX "IDX_user_report_user_id" ON "user_report" ("userId") `);
```

Comment:
```suggestion
        await queryRunner.query(`CREATE TABLE IF NOT EXISTS "user_report" ("reportedUserId" text NOT NULL, "userId" character varying(36) NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "reason" character varying(36) NOT NULL, "note" text, CONSTRAINT "PK_f804f389ba2f3ac0313080a1a82" PRIMARY KEY ("reportedUserId", "userId"))`);
        await queryRunner.query(`CREATE INDEX IF NOT EXISTS "IDX_user_report_reported_user_id" ON "user_report" ("reportedUserId") `);
        await queryRunner.query(`CREATE INDEX IF NOT EXISTS "IDX_user_report_user_id" ON "user_report" ("userId") `);
```

I know it's a brand new table, so chances that these are already existing is low, but just a best practice safety measure imho